1-js/99-js-misc/04-reference-type/2-check-syntax/task.md

```diff
@@ -1,10 +1,10 @@
-importance: 2
```

Comment:
You shouldn't translate metadata, please read the [Readme](https://github.com/javascript-tutorial/fa.javascript.info?tab=readme-ov-file#metadata).

Reply:
the metadata is translated. check the file. i wrote 'میزان سختی سوال' instead of 'importance'.

Reply:
> the metadata is translated. check the file. i wrote 'میزان سختی سوال' instead of 'importance'.

I said you **shouldn't**.

متادیتا نباید ترجمه بشه. لطفا لینکی که دادم رو مطالعه کنید.